simplq/src/components/pages/Admin/TokenList.jsx

```diff
@@ -1,19 +1,21 @@
 import React from 'react';
 import LoadingIndicator from 'components/common/LoadingIndicator';
+import { selectActiveTokens } from 'store/queue';
+import { useSelector } from 'react-redux';
 import Token from './Token';
 import styles from './admin.module.scss';
 
-function TokenList(props) {
-  const queueId = props.queueId;
+function TokenList({ queueId }) {
   let listContent = null;
+  const tokens = useSelector(selectActiveTokens);
 
-  if (!props.tokens) {
+  if (!tokens) {
```

Comment:
Tokens should always be at least an empty string. We have `LoadingStatus` component which can wrap tokens display and depend on `getActiveTokens` action. See `JonPage` for an exapmple.